packages/core/admin/admin/src/StrapiApp.tsx

```diff
@@ -133,6 +135,15 @@ class StrapiApp {
     this.createHook(MUTATE_EDIT_VIEW_LAYOUT);
 
     this.router = new Router(getInitialRoutes());
+
+    if (window.strapi.future.isEnabled('unstableWidgetsApi')) {
+      /**
+       * TODO:
+       * Instantiate this above with the other properties
+       * whe removing the future flag
```

Comment:
```suggestion
       * when removing the future flag
```

Reply:
I think we could instantiate it already, as long as we don't query the widgets store in the frontend I don't think it's an issue to have "future" code not under future flag as long it doesn't impact real app usage